flocker/node/functional/test_docker.py

```diff
@@ -641,6 +644,28 @@ def test_restart_policy_on_failure_maximum_count(self):
         d.addCallback(self.assertIn, ("5", "6"))
         return d
 
+    def test_command_line(self):
+        """
+        A container with custom command line is run with those arguments.
+        """
+        directory = FilePath(self.mktemp())
+        directory.makedirs()
+        d = self.start_container(
+            random_name(), image_name=u"busybox",
```

Comment:
A couple things:

  * I'm coming around to suspect that sharing any container images between tests (or between tests and real-world use) may result in more fragile tests.  Better that the test is the only thing that ever touches the image, otherwise who knows what crazy thing will happen inside Docker?  (Of course, if this is a valid line of thinking, there are other non-test implications that I am studiously not thinking about.)
  * `busybox` doesn't have an entrypoint so I don't think this demonstrates that the image's entrypoint is respected - which is the intended behavior, right?

Both of these suggest creating a tiny new image (in the spirit of #1281, sadly not yet merged) and using it in this test could be a good idea.